src/components/CardPlaylist/CardPlaylist.css

```diff
@@ -0,0 +1,37 @@
+.card-playlist {
+    height: fit-content;
+    width: 17rem; 
+}
+
+.container-img-card-playlist {
+    width: 17rem;
+    height: 11rem;
+    position: relative;
+}
+
+.img-card-playlist {
+    width: inherit;
+    height: inherit;
+}
+
+.container-overlay-img-playlist-card {
+    width: 6rem;
+    height: 11rem;
+    position: absolute;
+    top: 0;
+    right: 0;
+    background-color: #85828280;
```

Comment:
use color variable

Reply:
Will change it. Thanks for pointing this out